SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/CoordinatorConnection.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+using System;
+using System.Threading;
+using Improbable.Worker;
+
+namespace Improbable.WorkerCoordinator
```

Comment:
Could there be a nice little comment paragraph describing what the WorkerCoordinator does at the top of this file?

Reply:
Actually maybe not here. Is there a place that describes what the WorkerCoordinator does?